### ISSUE TITLE
Ohio county and township routes

### ISSUE THREAD

**Issue by 1ec5** [kennykb/osm-shields#3]
[This call to action](https://github.com/kennykb/osm-shields/wiki/Adding-new-networks#networks-most-needed) lists several [Ohio county](https://commons.wikimedia.org/wiki/Category:Diagrams_of_Ohio_county_route_markers) and [township](https://commons.wikimedia.org/wiki/Category:Diagrams_of_Ohio_township_route_markers) route networks. Here are SVG templates at Wikimedia Commons that cover these route networks. Each SVG currently has `00` or `000` as the route number, but this text would be substituted for `%num%`.

Locality | Network | Template | Modifier
----|----|----|---
Lucas Co. | `US:OH:LUC` | `US:county` | `LUCAS`
Bath Twp., Summit Co. | `US:OH:SUM:Bath` | ?
Medina Co. | `US:OH:MED` | `US:county:square:blue` | `MEDINA`
Stark Co. | `US:OH:STA` | `US:county` | `STARK`
Sharon Twp., Medina Co. | `US:OH:MED:Sharon` | `US:township:square:green` (like `US:county:square:green`) | `SHARON`
Carroll Co. | `US:OH:CAR` | `US:county:square` | `CARROLL`
Vinton Co. | `US:OH:VIN` | [Vinton County OH template.svg](https://commons.wikimedia.org/wiki/File:Vinton_County_OH_template.svg)
Ashland Co. | `US:OH:ASD` | [Ashland County OH template.svg](https://commons.wikimedia.org/wiki/File:Ashland_County_OH_template.svg)
Portage Co. | `US:OH:POR` | ?
Findlay Co. | `US:OH:FAY` | ?
Chester Twp., Wayne Co. | `US:OH:WAY:Chester` | ?
Plain Twp., Wayne Co. | `US:OH:WAY:Plain` | ?
Harrisville Twp., Medina Co. | `US:OH:MED:Harrisville` | `US:township:square` | `HARRISVILLE`
Congress Twp., Wayne Co. | `US:OH:WAY:Congress` | ?
Gilead Twp., Morrow Co. | `US:OH:MRW:Gilead` | ?
Union Co. | `US:OH:UNI` | `US:county:square` | `UNION`
Westfield Twp., Medina Co. | `US:OH:MED:Westfield` | ?
Jefferson Twp., Logan Co. | `US:OH:LOG:Jefferson` | `US:township:square` | `JEFFERSON`
Litchfield Twp., Medina Co. | `US:OH:MED:Litchfield` | ?
Seneca Co. | `US:OH:SEN` | `US:county` | `SENECA`
Wadsworth Twp., Medina Co. | `US:OH:MED:Wadsworth` | `US:township:square` | `WADS`
Wood Co. | `US:OH:WOO` | ?
Stokes Twp., Logan Co. | `US:OH:LOG:Stokes` | ?
Licking Co. | `US:OH:LIC` | `US:county:square` | `LICKING`
Union Twp., Logan Co. | `US:OH:LOG:Union` | ?
Washington Twp., Logan Co. | `US:OH:LOG:Washington` | ?
Harrison Twp., Logan Co. | `US:OH:LOG:Harrison` | ?
Athens Co. | `US:OH:ATH` | `US:county:square:green` | `ATHENS`
Chatham Co., Medina Co. | `US:OH:MED:Chatham` | ?
McArthur Twp., Logan Co. | `US:OH:LOG:McArthur` | ?
Spencer Twp., Medina Co. | `US:OH:MED:Spencer` | ?
Rushcreek Twp., Logan Co. | `US:OH:LOG:Rushcreek` | ?
Clinton Twp., Wayne Co. | `US:OH:WAY:Clinton` | ?
Richland Twp., Logan Co. | `US:OH:LOG:Richland` | ?
Zane Twp., Logan Co. | `US:OH:LOG:Zane` | ?
Miami Twp., Logan Co. | `US:OH:LOG:Miami` | ?
Perry Twp., Logan Co. | `US:OH:LOG:Perry` | ?
Fremont, Sandusky Co. | `US:OH:SAN:Fremont` | [Fremont, Ohio, city route shield template.svg](https://commons.wikimedia.org/wiki/File:Fremont,_Ohio,_city_route_shield_template.svg)
Lake Twp., Logan Co. | `US:OH:LOG:Lake` | ?
Homer Twp., Medina Co. | `US:OH:MED:Homer` | ?
Lorain Co. | `US:OH:LOR` | ?
Boston Twp., Summit Co. | `US:OH:SUM:Boston` | ?
Bloomfield Twp., Logan Co. | `US:OH:LOG:Bloomfield` | ?
Chippewa Twp., Wayne Co. | `US:OH:WAY:Chippewa` | ?
Franklin Twp., Morrow Co. | `US:OH:MRW:Franklin` | ?
Ashland Co. township roads | `US:OH:ASD:TWP` | [Ashland County OH template.svg](https://commons.wikimedia.org/wiki/File:Ashland_County_OH_template.svg) | *
Wooster Twp., Wayne Co. | `US:OH:WAY:Wooster` | ?
Guilford Twp., Medina Co. | `US:OH:MED:Guilford` | ?
Canaan Twp., Wayne Co. | `US:OH:WAY:Canaan` | ?
Berlin Twp., Holmes Co. | `US:OH:HOL:Berlin` | ?
Fairfield Twp., Columbiana Co. | `US:OH:COL:Fairfield` | ?
Bokescreek Twp., Logan Co. | `US:OH:LOG:Bokescreek` | ?
Montville Twp., Medina Co. | `US:OH:MED:Montville` | ?
Hamilton Co. | `US:OH:HAM` | —
Hardin Co. | `US:OH:HAR` | `US:county:square:green` | `HARDIN`
Paint Twp., Wayne Co. | `US:OH:WAY:Paint` | ?
Unity Twp., Columbiana Co. | `US:OH:COL:Unity` | ?
West Twp., Columbiana Co. | `US:OH:COL:West` | ?
Jackson Twp., Monroe Co. | `US:OH:MOE:Jackson` | ?
Coshocton Co. | `US:OH:COS` | `US:county:square` | `COSHOCTON`
Wyandot Co. | `US:OH:WYA` | `US:county:square` | `WYANDOT`

In general, you can find Wikimedia Commons shields and shield templates through the [modules that power road-related infoboxes](https://en.wikipedia.org/wiki/Module:Road_data/strings/USA/OH) at the English Wikipedia.

Previously [bug 1025086](https://bugs.launchpad.net/osm-shields/+bug/1025086) at Launchpad.

**Comment by kennykb** [comment]
Hmm, you've made it farther down the list than I posted; did you actually run 'routeGraphics.tcl' successfully?  (If so, thanks for following so assiduously!)

I think Vinton County should maybe be named US:county:square:yellow - I suspect that's not the last we'll see of that particular format. (According to the original ticket, Trumbull County uses it as well?)

Ashland County and Fremont City don't appear to render correctly in Wikipedia's PNG's, or in the browser when I look at the SVG template there.  I'll have to see whether it's any different in Inkscape (plus, we're eventually going to want Mapnik's built-in SVG interpreter to be able to render all of these).

Thanks for the research!

**Comment by 1ec5** [comment]
I haven’t run routeGraphics.tcl yet; I just went through taginfo.

Vinton County’s signage is… special. It does look similar to Trumbull County’s, but the font is different and the township name is missing. (At the size displayed in this renderer, I’d omit the township name from Trumbull County’s shields anyways.)

Ashland County’s template does render correctly in Inkscape. I think it behaves weirdly because of vertical text. We can apply a rotation transform instead to achieve the same effect.

**Comment by kennykb** [comment]
Ok, as far as Ohio goes, I think I've scraped as much from Wikipedia as I'm going to get.

I've got all the counties, cities and townships that you gave links for, plus a couple you didn't.  I also got all ten Canadian provinces done (plus Yukon and Northwest Territories) and the Mexican state of Quéretaro. I'm down mostly to networks that I suspect should be unsigned_refs, plus networks with only a couple of  dozen or fewer routes in them. If there isn't enough takeup of this project for other people to work on the onesie-twosies, then it's not worth doing.

Feel free to open another ticket if there's something specific and actionable remaining to do.
